packages/cda-validator/README.md

```diff
@@ -1,21 +1,15 @@
-# cda-core-2.0
+## CDA Validator
 
-Resources for Clinical Document Architecture (CDA) release 2.0
+This is a clone of the [CDA-validator](Original repository: https://github.com/HL7/CDA-core-2.1), which contains only the schema and extensions components of the original repo.
```

Comment:
invalid markdown link on `CDA-validator` - it should only have one display text and one link, something like:

```
[CDA-validator](https://github.com/HL7/CDA-core-2.1)
```

or

```
CDA-validator - [original repository](https://github.com/HL7/CDA-core-2.1)
```